playwright-tests/utils/actions.ts

```diff
@@ -0,0 +1,52 @@
+import { expect, Page } from '@playwright/test';
+import { DataTestIdsEnum } from '../../src/localConstants/dataTestIds.enum';
+import {
+  AccountStatesEnum,
+  GlobalDataEnum,
+  LoginFilesEnum
+} from '../utils/enums.ts';
+
+export const confirmPem = async ({ page, file = LoginFilesEnum.pem }) => {
+  await page.getByText('Select a file').click();
+  await page.setInputFiles(DataTestIdsEnum.inputFile, file);
+  await page.getByTestId(DataTestIdsEnum.submitButton).click();
+};
+export const login = async ({ page, file = LoginFilesEnum.pem, user = '' }) => {
+  // Wait for and click the access wallet button
+
+  if (file !== LoginFilesEnum.pem) {
+    await page.getByTestId(DataTestIdsEnum.keystoreBtn).click();
+    await page.setInputFiles(DataTestIdsEnum.inputFile, file);
+    await page.getByTestId(DataTestIdsEnum.accessPass).fill('Develop13#');
+    console.log(user);
+    await page.getByTestId(DataTestIdsEnum.submitButton).click();
+    user ? await page.getByTestId(`check_${user}`).click() : '';
+    await page.waitForTimeout(1000);
+    await page.getByTestId(DataTestIdsEnum.confirmBtn).click();
+    await page.waitForTimeout(2000);
+  } else {
+    await page.getByTestId(DataTestIdsEnum.pemBtn).click();
+    await confirmPem({ page, file });
+  }
+};
+
+export const handlePopup = async (page: Page, triggerPopupAction) => {
```

Comment:
do not export multiple functions from a file
one function per file and file has function name